docs/kf.dev/content/en/docs/operators/administering-kf/setting-up-a-custom-domain.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Setting up a custom domain"
+weight: 55
+type: "docs"
+---
+
+By default, Kf apps are accessible on domains exposed by Knative Serving.
+Kf extends Knative Serving's domain system to allow path based routing, per-app domains, wildcard domains, and other routing services.
```

Comment:
Unless I'm mistaken, I don't think we allow wildcard domains.

Reply:
you're right, that should be hosts

Reply:
I don't know that we really support that either. I don't think I can have a wildcard directed to a specific app.

Reply:
I tested that back when you originally added routing and it worked alright then.